docs/src/main/sphinx/release/release-417.md

```diff
@@ -0,0 +1,40 @@
+# Release 417 (10 May 2023)
+
+## General
+
+* Improve performance of `UNION ALL` queries. ({issue}`17265`)
+* Fix over-reading data in the `TEXTFILE` decoder for structs that contain more
+  columns in the file than are mapped in the schema. ({issue}`17364`)
+
+## Delta Lake connector
+
+* Add support for [`COMMENT ON VIEW`](/sql/comment). ({issue}`17089`)
+* Improve efficiency of packing data in files written by the Parquet writer. ({issue}`17373`)
+* Improve the layout of data in Parquet files, leading to faster reads. ({issue}`17404`)
+* Improve read performance for tables with `ROW` or `STRUCT` columns when only a
```

Comment:
Are `ROW` and `STRUCT` supposed to be Delta Lake data types? Trino doesn't have a `STRUCT` data type.

Also, lower-case the type names when referring to Trino types: `row`

Reply:
Hm.. in the type mapping and others docs and code we are mostly using all uppercase for data types..we should probably standardize